apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -403,6 +406,13 @@ function AddedSectionsGrid() {
         return scheduleNames[scheduleIndex];
     }, [scheduleNames, scheduleIndex]);
 
+    // "No Courses Added Yet!" notification
+    const noCoursesNotif = (
```

Comment:
```suggestion
    const NoCoursesBox = (
```
Bit of a nit, but "Notif" might be the wrong way to phrase this element